packages/turndown/src/turndown.js

```diff
@@ -239,39 +215,35 @@ function replacementForNode (node) {
   var content = process.call(this, node, rule.escapeContent ? rule.escapeContent() : 'auto')
   var whitespace = node.flankingWhitespace
   if (whitespace.leading || whitespace.trailing) content = content.trim()
+
+  const replaceNonbreakingSpaces = space => {
+    // \u{00A0} is a nonbreaking space
+    return space.replace(/\u{00A0}/ug, this.options.nonbreakingSpace);
+  };
```

Comment:
I added this function after merging changes to optionally replace `\u00A0` characters with `&nbsp;`, making non-breaking spaces noticeable within the markdown editor.